src/components/timeline/ActiveTabResourceTrack.js

```diff
@@ -58,22 +58,26 @@ class ActiveTabResourceTrackComponent extends PureComponent<Props, State> {
     };
   }
 
-  _onMouseUp = (event: MouseEvent) => {
+  _onMouseUp = fromWhere => (event: MouseEvent) => {
```

Comment:
I think we can name this something like `clickedArea` and it would be better if we can give that paramater a type. Its type can be something like `'timelineTrackResourceLabel' | 'timelineTrackRow'`.